### DISCUSSION TITLE
Loading previously downloaded directories

### DISCUSSION THREAD

**Discussion by Pihg** [shadowmoose/RedditDownloader#208] [Q&A]
I’ve recently moved from Windows to unRAID (Slackware based), and have setup the RMD-ubuntu precompiled binary to run as I need it to. Is there a way for me to have it read the download directory from when it was on windows and sort of pick up where it left off?

**Comment by shadowmoose** [comment]
Yes, RMD should work cross-platform if you copy everything over (including the manifest database file) and run within the same relative directory layout.

**Reply by Pihg** [reply]
I had a feeling as such, it seems I need to tinker the settings more. Does it make any difference if windows had it on version 3.0.x and the new one is 3.1.5?

Edit: disregard. My error was I simply specified the folder that the manifest was in, and not the full path to the manifest file explicitly. Thanks for the help!
